### DISCUSSION TITLE
How to change the language?

### DISCUSSION THREAD

**Discussion by DeeKey** [OpenHantek/OpenHantek6022#234] [Q&A] [answered]
Seems currently language is system dependent. 
How to change it without changing system settings?

**Comment by Ho-Ro** [comment, marked as answer]
If you're on a Linux system you could type `LANG=fr Openhantek` and you get the French localisation (or LANG=de for German etc...), or type `OpenHantek -i` (or `OpenHantek --international`) to get the original English (international) language. 
If you're on Windows or Mac `-i` will work - for other languages - I don't know.
